.pre-commit-config.yaml

```diff
@@ -0,0 +1,55 @@
+ci:
+  autofix_prs: false
+
+repos:
+  # Lint first (with safe autofixes)
+  - repo: https://github.com/astral-sh/ruff-pre-commit
+    rev: v0.6.9
+    hooks:
+      - id: ruff
+        name: ruff-check
+        args: [--fix, --show-fixes]
+        files: ^(kubeflow-pipelines/(docling-standard|docling-vlm)/.*\.py|[^/]+\.py)$
```

Comment:
Do we need to have `files` here? I know we're going to add `scripts` and also python files in notebooks as well.

Reply:
I'm changing this to `types_or: [python]` and `types: [jupyter]` so that all .py and notebook files anywhere in the repo get picked up automatically and then we won't need to manually expand regexes every time.